### PR TITLE
chore: Update PR template to match statechannels repo

### PR DESCRIPTION
**Pull request by wunnaii**
# Description
Update PR template with hyperlink to zenhub pipeline definition.

# Checklist:

## Code quality
- [x] I have written clear commit messages
- [ ] I have performed a self-review of my own code
- [x] I have scoped this change as narrowly as possible
- [ ] I have separated logic changes from refactor changes (formatting, renames, etc.)
- [ ] I have commented my code wherever necessary
- [ ] I have added tests that prove my fix is effective or that my feature works
## Project management
- [x] I have applied the [appropriate labels](https://www.notion.so/Team-working-agreements-2a95c926bb5642e5a5c42e4b74a9dd24#b304e56734a74dfbb341b8b4b27b1c0c)
- [ ] I have linked to relevant issues
- [ ] I have added dependent tickets
- [x] I have assigned myself to this PR
- [x] I have chosen the appropriate pipeline

### CODE REVIEW

**Review by andrewgordstewart — APPROVED** [review]
Looks like a copy/paste here?

**Comment by wunnaii** [comment]
yeah forgot i was allowed to merge template updates into the remaining repos without approval, after the first template is approved.

**Pull request merged**
